client/app/routes/create-account.tsx

```diff
@@ -84,6 +124,12 @@ export default function CreateAccount() {
 										{emailState.error.message}
 									</small>
 								)}
+								{actionData?.type === "conflict" &&
```

Comment:
We should only show one error at a time. 

Currently, if the email is invalid and also came back as already existing, we’ll see two errors.